.github/workflows/continuous-integration.yml

```
@@ -205,16 +205,42 @@ jobs:
         id: get-broken-link-info
         run: node ./script/github-actions/check-broken-links.js ${{ matrix.buildtype }}
 
-      - name: Compress ${{ matrix.buildtype }} build
-        if: ${{ github.ref == 'refs/heads/master' && success() }}
-        run: tar -zcvf ${{ matrix.buildType }}.tar.gz build
+      - name: Generate build details
+        run: |
+          cat > build/${{ matrix.buildtype }}/BUILD.txt << EOF
+          BUILDTYPE=${{ matrix.buildtype }}
+          BRANCH_NAME=$(echo "${GITHUB_REF#refs/heads/}")
+          RUN_ID=${{ github.run_id }}
+          REF=${{ github.sha }}
+          BUILDTIME=$(date +%s)
+          EOF
+
+      - name: Compress vagovprod build for E2E
+        if: ${{ github.ref == 'refs/heads/master' && matrix.buildType == 'vagovprod' }}
+        run: tar -jcvf test-build.tar.bz2 build/vagovprod
+
+      - name: Upload build for E2E
+        uses: actions/upload-artifact@v2
+        if: ${{ github.ref == 'refs/heads/master' && matrix.buildType == 'vagovprod' }}
+        with:
+          name: test-build.tar.bz2
+          path: content-build/test-build.tar.bz2
+          retention-days: 1
 
-      - name: Upload build
+      - name: Prearchive
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: node ./script/prearchive.js --buildtype=${{ matrix.buildtype }}
+
+      - name: Compress ${{ matrix.buildtype }} build w/ urls updated
```

Not sure if prearchive will always just be updating URLs. Can keep it general, I suppose.

```suggestion
      - name: Compress prearchived ${{ matrix.buildtype }} build
```